src/v2/guide/migration.md

```diff
@@ -466,14 +467,12 @@ new Vue({
 </script>
 {% endraw %}
 
-Using the `debounce` attribute, there'd be no way to detect the "Typing" state, because we lose access to the input's real-time state. By decoupling the debounce function from Vue however, we're able to debounce only the operation we want to limit, removing the limits on features we can develop:
+Menggunakan atribut `debounce`, tidak akan ada cara untuk mengenali "Tipe" *state*, karena kita kehilangan akses pada *input real-time state*. Namun memisahkan fungsi *debounce* dari Vue, kami hanya dapat *debounce* pada operasi yang kami ingin batasi, menghapus keterbatasan pada fitur dapat dikembangkan sebagai berikut:
```

Comment:
```suggestion
Menggunakan atribut `debounce`, tidak ada cara untuk mengenali "Tipe" *state*, karena kita kehilangan akses pada *input real-time state*. Namun dengan memisahkan fungsi *debounce* dari Vue, kita hanya dapat *debounce* pada operasi yang kita ingin batasi, menghapus keterbatasan pada fitur dapat dikembangkan sebagai berikut:
```

Reply:
Dalam dokumentasi biasanya, 'we' mengacu pada 'kita' untuk mencontohkan. Tergantung pada penempatan kalimat.

Reply:
sudah diperbaiki